included_software.md

```diff
@@ -10,6 +10,7 @@ The specific patch versions included will depend on when the image was last buil
 
 * Ruby - `RUBY_VERSION`, `.ruby-version`
   * 2.6.2 (default)
+  * 2.7.2
```

Comment:
@ingride Just checking whether 2.7.2 is meant to be the new default here instead of an addition.

Reply:
it's meant to be a new addition, many customers depend on 2.6.2 which is why it's default.